src/zkp/mod.rs

```diff
@@ -33,6 +33,7 @@ pub(crate) trait ProofContext {
     fn as_bytes(&self) -> Vec<u8>;
 }
 
+#[cfg(test)]
 /// This is a temporary preliminary implementation that will be modified by
 /// issue #240.
```

Comment:
nit: This is now outdated! Maybe change this to a line recommending against use in an actual protocol in case someone is tempted to remove the `#[cfg(test)]` later? Or just remove.